### DISCUSSION TITLE
Welcome to Topaz Discussions!

### DISCUSSION THREAD

**Discussion by tbepler** [tbepler/topaz#75] [General]
## 👋 Welcome to Topaz discussions!
  We’re using Discussions as a place to connect with other members of our community, to ask and answer questions, seek help, and otherwise discuss Topaz topics that are not bug reports/feature requests. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by A-Talavera** [comment]
Hello,
I am using Topaz and I am having two main issues.
1. I can get the particles coordinates after running a topaz_extract with the resnet8_u64 pretrained model. It finds close to 10^6 particles, then I converted the coordinates_output.txt file into .csv format. The problems comes that after importing the .csv fine into Topaz GUI it does not show up the particles. Do you have an idea of what could be causing this problem?
2. This issue it is even more important to me. I am running Topaz as a stand alone program, and I want to import the either the coordinates file or the particles into Relion. With both I always get a complaint from Relion 3.1 and failed to import the job. This complaint is always related to file formats problems.
    -To convert to particle stack I use:
          topaz particle_stack --image_root=/path/to/MotionCorr/Job/Movies --size 200 --threshold=0 -- 
          metadata=/pathe/to/CtfFind/job/micrographs_ctf.star -/path/to/predicted_particles_stack.mrc /path/to/extracted_coordinates.txt
    -In Relion 3.1, 
           - If I tray to Import the  predicted_particles_stack.mrc as Node type: Particles star file I get this error message:
                 ERROR: not all necessary variables defined in _optics.star file: rlnPixelSize, rlnVoltage and rlnSphericalAberration. Make sure 
                 to convert older STAR files anew in version-3.1, with relion_convert_star.
          - If I tray to Import the coordinates file Extract.star (previously converted with Topaz) as Node type: Particles coordinates
                 ERROR: Import: Cannot write file: Import/job085/coords_suffixTopaz/NoDenoise/Extract_01.star
          - If I try to Extract (using the option re-extract refined particles) using the predicted_particles_stack.star
               ERROR: A line in the STAR file contains fewer columns than the number of labels. Expected = 17 Found = 5 application called 
               MPI_Abort(MPI_COMM_WORLD, 1) - process 13
           - In general I have tried a lot of different ways but I have always failed to get it to work in relion
           - Nevretheless, I can import the particles into CryoSparc
To make the story short, Is it a way to smoothly import  into Relion the coordinates and/or particles extracted with stand-alone Topaz?

Thanks a lot in advanced.
Best regards,
Ariel

**Reply by Guillawme** [reply]
Hi,

Have you tried to `convert older STAR files anew in version-3.1, with relion_convert_star` as suggested by RELION? If so, did it not work?

> To make the story short, Is it a way to smoothly import into Relion the coordinates and/or particles extracted with stand-alone Topaz?

Other than fixing the generated star file, I don't know. You can wait for RELION-4, which will have a better integration of Topaz.

Next time you post about a specific issue, please open a new discussion: it makes the discussions section much easier to search for solutions, in case somebody in the future runs into the same problem.
